char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
+lowecase_chars = ['countat1','a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
```

Comment:
also don't need this at all for a couple of reasons.
1. a string is a list of characters already so an easier way to represent this might be ```lowercase_chars = " abcdefghijklmnopqrstuvwxyz"```
2. the string module already has string.ascii_lowercase which is identical without the padding.
3. I think though the best thing to leverage here is the built in [`ord()` ](https://docs.python.org/3/library/functions.html?highlight=ord#ord)method
```suggestion
```